mapie/quantile_regression.py

```diff
@@ -453,6 +486,11 @@ def predict(  # type: ignore
             - [:, 1, :]: Upper bound of the prediction interval.
         """
         check_is_fitted(self, self.fit_attributes)
+        if alpha is not None:
```

Comment:
Shall we raise a warning instead by saying that the `alpha` defined in `predict` won't be used for PI estimates and then fix this `alpha` to None ?

Reply:
I think leaving a warning is indeed fine, otherwise, I would not bother to change the value necessarily since we get the `alpha` later from `self.alpha`.